src/seml/database.py

```diff
@@ -18,6 +20,44 @@ def get_collection(collection_name, mongodb_config=None, suffix=None):
     return db[collection_name]
 
 
+def export_collection(collection_name, path=None):
+    collection = get_collection(collection_name)
+    cursor = collection.find({})
+    if path == None:
+        path = f'{collection_name}.bson'
+    with open(path, 'w') as file:
+        file.write(dumps(cursor))
+    logging.info(
```

Comment:
I'd print info that this doesn't contain source files and is not meant for running experiments/cannot run experiments.